oscarapi/basket/operations.py

```diff
@@ -57,7 +57,7 @@ def prepare_basket(basket, request):
 
 def get_basket(request, prepare=True):
     "Get basket from the request."
-    if request.user.is_authenticated():
+    if request.user.is_authenticated:
```

Comment:
Isn't this a fix for a really bad bug? How could this ever have worked?

Reply:
Using User.is_authenticated() and User.is_anonymous() as methods rather than properties is no longer supported in Django 2.

Reply:
@specialunderwear See https://docs.djangoproject.com/en/2.0/releases/2.0/#features-removed-in-2-0

Reply:
> How could this ever have worked?

In Django 1.11.13 `is_authenticated` is a property returns [CallableBool](https://github.com/django/django/blob/1.11.13/django/utils/deprecation.py#L87), so it could be used like `user.is_authenticated()` with the same meaning as `user.is_authenticated`.
https://github.com/django/django/blob/1.11.13/django/contrib/auth/base_user.py#L96
https://github.com/django/django/blob/1.11.13/django/contrib/auth/models.py#L448